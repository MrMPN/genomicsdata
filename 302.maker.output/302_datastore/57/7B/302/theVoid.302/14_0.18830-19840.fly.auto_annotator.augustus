# This output was generated with AUGUSTUS (version 2.5.5).
# AUGUSTUS is a gene prediction tool for eukaryotes written by Mario Stanke (mario.stanke@uni-greifswald.de)
# and Oliver Keller (keller@cs.uni-goettingen.de).
# Please cite: Mario Stanke, Mark Diekhans, Robert Baertsch, David Haussler (2008),
# Using native and syntenically mapped cDNA alignments to improve de novo gene finding
# Bioinformatics 24: 637-644, doi 10.1093/bioinformatics/btn013
# reading in the file /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/14_0.18830-19840.fly.auto_annotator.xdef.augustus ...
# Setting 1group1gene for E.
# Sources of extrinsic information: M P E 
# Have extrinsic information about 1 sequences (in the specified range). 
# Initialising the parameters ...
# fly version. Use default transition matrix.
# Looks like /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/14_0.18830-19840.fly.auto_annotator.augustus.fasta is in fasta format.
# We have hints for 1 sequence and for 1 of the sequences in the input set.
#
# ----- prediction on sequence number 1 (length = 1010, name = 302) -----
#
# Constraints/Hints:
302	EST-EXON	exonpart	204	224	0	-	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-EXON	exonpart	284	514	0	-	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-EXON	exonpart	580	810	0	-	.	grp=n_peices;src=E "10;0.995;0:0"
302	EST-INTRON	intronpart	226	282	0	-	.	grp=n_pieces;src=E "1;1;0:0"
302	EST-INTRON	intronpart	516	578	0	-	.	grp=n_pieces;src=E "1;1;0:0"
302	PROTEIN	CDSpart	204	224	0	-	.	grp=p_pieces;src=P "100000;1;0:0"
302	PROTEIN	CDSpart	284	514	0	-	.	grp=p_pieces;src=P "35111.9;1;0:1"
302	PROTEIN	CDSpart	580	810	0	-	.	grp=p_pieces;src=P "100000;1;0:0"
302	EST-EXON	genicpart	225	283	0	.	.	grp=n_peices;src=E "2;1;0:0"
302	EST-INTRON	genicpart	283	515	0	.	.	grp=n_pieces;src=E "2;1;0:0"
302	EST-EXON	genicpart	515	579	0	.	.	grp=n_peices;src=E "2;1;0:0"
# Predicted genes for sequence number 1 on reverse strand
# start gene g1
302	AUGUSTUS	gene	196	810	1.91	-	.	g1
302	AUGUSTUS	transcript	196	810	0.91	-	.	g1.t1
302	AUGUSTUS	stop_codon	196	198	.	-	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	225	282	0.97	-	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	intron	515	582	1	-	.	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	196	224	0.91	-	2	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	283	514	0.97	-	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	CDS	583	810	1	-	0	transcript_id "g1.t1"; gene_id "g1";
302	AUGUSTUS	start_codon	808	810	.	-	0	transcript_id "g1.t1"; gene_id "g1";
# protein sequence = [MADTQAKPAASKAAKAPKTAKAPKAAKAPKAEKAVATETKVSAKKYKRHGRLFAKAVFTGYKRGLRNQHENHAILKIE
# GARRKEHGQFYVGKRCVYVYKAQTKKCVPQHPDRKTRVRAVWGKVTRIHGNTGSVRARFNRNLPGHAMGHRIRIVNAVPIKDLS]
# Evidence for and against this transcript:
# % of transcript supported by hints (any source): 80
# CDS exons: 2/3
#      E:   2 
#      P:   2 
# CDS introns: 2/2
#      E:   2 
# 5'UTR exons and introns: 0/0
# 3'UTR exons and introns: 0/0
# hint groups fully obeyed: 1
#      E:   1 (n_pieces)
# incompatible hint groups: 2
#      E:   1 (n_peices)
#      P:   1 (p_pieces)
302	AUGUSTUS	transcript	201	810	1	-	.	g1.t2
302	AUGUSTUS	stop_codon	201	203	.	-	0	transcript_id "g1.t2"; gene_id "g1";
302	AUGUSTUS	intron	225	286	1	-	.	transcript_id "g1.t2"; gene_id "g1";
302	AUGUSTUS	intron	515	582	1	-	.	transcript_id "g1.t2"; gene_id "g1";
302	AUGUSTUS	CDS	201	224	1	-	0	transcript_id "g1.t2"; gene_id "g1";
302	AUGUSTUS	CDS	287	514	1	-	0	transcript_id "g1.t2"; gene_id "g1";
302	AUGUSTUS	CDS	583	810	1	-	0	transcript_id "g1.t2"; gene_id "g1";
302	AUGUSTUS	start_codon	808	810	.	-	0	transcript_id "g1.t2"; gene_id "g1";
# protein sequence = [MADTQAKPAASKAAKAPKTAKAPKAAKAPKAEKAVATETKVSAKKYKRHGRLFAKAVFTGYKRGLRNQHENHAILKIE
# GARRKEHGQFYVGKRCVYVYKAQTKKCVPQHPDRKTRVRAVWGKVTRIHGNTGSVRARFNRNLPGHAMGHRIRIMLYPSRI]
# Evidence for and against this transcript:
# % of transcript supported by hints (any source): 60
# CDS exons: 1/3
#      E:   1 
#      P:   1 
# CDS introns: 2/2
#      E:   2 
# 5'UTR exons and introns: 0/0
# 3'UTR exons and introns: 0/0
# hint groups fully obeyed: 1
#      E:   1 (n_pieces)
# incompatible hint groups: 2
#      E:   1 (n_peices)
#      P:   1 (p_pieces)
# end gene g1
###
# command line:
# /home/genomica/Programs/maker/exe/augustus/bin/augustus --species=fly --strand=backward --UTR=off --hintsfile=/home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/14_0.18830-19840.fly.auto_annotator.xdef.augustus --extrinsicCfgFile=/home/genomica/Programs/maker/exe/augustus/config/extrinsic/extrinsic.MPE.cfg /home/genomica/genomicsdata/302.maker.output/302_datastore/57/7B/302//theVoid.302/14_0.18830-19840.fly.auto_annotator.augustus.fasta
