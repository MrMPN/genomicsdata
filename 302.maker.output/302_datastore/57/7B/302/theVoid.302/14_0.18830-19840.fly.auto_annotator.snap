>302 14_0 offset:18830
Einit	583	810	-	64.614	0	0	2	302-snap.1
Exon	287	514	-	66.723	0	0	1	302-snap.1
Eterm	201	224	-	16.294	0	0	0	302-snap.1
